auctions/models.py

```diff
@@ -184,22 +211,47 @@ def __unicode__(self):
 
 
 @receiver(post_save, sender=Vote)
-def notify_approved_claim(sender, instance, created, **kwargs):
+def update_claim_status(sender, instance, created, **kwargs):
     claim = instance.claim
-    votes_needed = Bid.objects \
-                      .filter(issue=claim.issue) \
-                      .exclude(user=claim.user) \
-                      .filter(offer__gt=0) \
-                      .count()
+    offers_needed = claim.offers.count()
 
-    approvals = Vote.objects \
-                    .filter(claim=claim, approved=True) \
-                    .exclude(user=claim.user) \
-                    .count()
+    if claim.num_votes > 0:
+        claim.status = "Pending"
+        claim.save()
 
-    # TODO: notify claimant when claim rejected
+    if claim.num_approvals == offers_needed:
+        claim.status = "Approved"
+        claim.save()
+
+    if offers_needed > 0:
+        if (claim.num_rejections / float(offers_needed)) >= 0.5:
+            claim.status = "Rejected"
+            claim.save()
+
+
+@receiver(post_save, sender=Vote)
+def notify_approved_claim(sender, instance, created, **kwargs):
```

Comment:
We may want to fold this into the `update_claim_status` signal handler? I'm not sure what the proper Djangonaut™ approach is here: multiple signal handlers for models, or a single signal handler.